website/next-env.d.ts

```diff
@@ -2,4 +2,4 @@
 /// <reference types="next/image-types/global" />
 
 // NOTE: This file should not be edited
-// see https://nextjs.org/docs/pages/building-your-application/configuring/typescript for more information.
+// see https://nextjs.org/docs/basic-features/typescript for more information.
```

Comment:
Total nitpick, but it looks like you're not running the version of Next that's defined in `pnpm-lock.yaml` since this comment got reverted. You should run `pnpm install` locally. 👍